Entities/Industry/Drill/Drill.as

```diff
@@ -43,7 +44,7 @@ void onInit(CSprite@ this)
 			anim.AddFrames(frames);
 		}
 		heat.SetAnimation(anim);
-		heat.SetRelativeZ(0.1f);
+		heat.SetRelativeZ(1.0f);
```

Comment:
What does this change? Document it in the PR if it wasn't accidentally left there

Reply:
i have no idea why this is here 

either was testing something and left it accidentally or it was changed to fix some obscure visual issue that was happening only on staging, but i have no idea what it could have been and don't know if i'm not misremembering